src/mainwindow/mainwindow.cpp

```diff
@@ -189,7 +190,7 @@ void MainWindow::setupMainMenu()
 
     // opening settings window
     QAction *pSettingsAction = toolsMenu->addAction("&Settings...", this, &MainWindow::onSettingsTriggered);
-    pSettingsAction->setDisabled(true);
+    //pSettingsAction->setDisabled(true);
```

Comment:
don't add commented lines of code to the commit. You can simply get this from previous commits when you need